.github/workflows/tag-new-release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release a new Armeria version
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release Version'
+        required: true
+        type: string
+      next_version:
+        description: 'Next Version'
+        required: true
+        type: string
+
+env:
+  LC_ALL: "en_US.UTF-8"
+
+jobs:
+  tag-new-version:
+    if: github.repository == 'line/armeria'
+    runs-on: self-hosted
+
+    steps:
+      - name: Validate inputs
+        run: |
+          if [[ ! "${{ inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: 'release_version' should be in SemVer format."
+            exit 1
+          fi
+          if [[ ! "${{ inputs.next_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: 'next_version' should be in SemVer format."
+            exit 1
+          fi
+
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
PAT is required to trigger another workflow. https://stackoverflow.com/a/60418414/1736581